generator/.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
+*.sh text eol=lf
+*.md text eol=lf
+*.tmpl text eol=lf
```

Comment:
@samsonkolge, any specific reason for including this change as part of this PR?
Did you face any issues with the git-checkin/checkout file endings? 

If not, let's remove this change from the PR.

Reply:
Yes I faced this error while committing. I though it will help others of the hassle.

Reply:
Could you please paste the error here?

This enforcing line ending might change a lot of other committed files too over time, & I would err on the side of not doing that unless absolutely required.

Reply:
`warning: LF will be replaced by CRLF in generator/annual-report/sig_report.tmpl.
The file will have its original line endings in your working directory`

Reply:
ok, then let's use the following (to avoid specifying each file type explicitly )

```suggestion
# Always check-out / check-in files with LF line endings.
* text=auto eol=lf
```

Reply:
git attributes change is not reflected in the latest commit. Please check!

Also, add a proper commit message something like `move list of generated list of KEPs in comments`